library/Icingadb/Widget/HostListItemDetailed.php

```diff
@@ -4,11 +4,75 @@
 
 namespace Icinga\Module\Icingadb\Widget;
 
+use Icinga\Application\Icinga;
+use Icinga\Module\Icingadb\Common\ListItemDetailedLayout;
 use Icinga\Module\Icingadb\Compat\CompatPluginOutput;
+use Icinga\Module\Icingadb\Util\Perfdata;
+use Icinga\Module\Icingadb\Util\PerfdataSet;
 use ipl\Html\BaseHtmlElement;
+use ipl\Html\Html;
+use ipl\Html\HtmlString;
+use ipl\Web\Widget\Icon;
+use ipl\Web\Widget\StateBall;
 
-class HostListItemDetailed extends HostListItem
+class HostListItemDetailed extends BaseHostListItem
 {
+    use ListItemDetailedLayout;
+
+    protected $pie;
+
+    protected function getStateBallSize()
+    {
+        return StateBall::SIZE_LARGE;
+    }
+
+    protected function assembleFooter(BaseHtmlElement $footer)
+    {
+        $this->pie = HtmlString::create('');
+
+        $statusIcons = Html::tag('div', ['class' => 'status-icons']);
```

Comment:
Please use `new HtmlElement('div', ['class' => 'status-icons'])` instead. The `Html` class shouldn't be used inside of widgets. (It seems to be a small optimization, but it makes a difference if it's called many times in a row. `Html::tag` is a detour after all.)

Reply:
That's not resolved, or am i blind?